lib/suspenders/generators/app_generator.rb

```diff
@@ -160,31 +159,29 @@ def install_bitters
     end
 
     def install_refills
-      say "Install Refills"
+      say 'Install Refills'
       build :install_refills
     end
 
     def setup_git
-      if !options[:skip_git]
-        say 'Initializing git'
-        invoke :setup_gitignore
-        invoke :init_git
-      end
+      return if options[:skip_git]
+      say 'Initializing git'
+      invoke :setup_gitignore
+      invoke :init_git
     end
 
     def create_heroku_apps
-      if options[:heroku]
-        say "Creating Heroku apps"
-        build :create_heroku_apps, options[:heroku_flags]
-        build :provide_review_apps_setup_script
-        build :set_heroku_serve_static_files
-        build :set_heroku_remotes
-        build :set_heroku_rails_secrets
-        build :create_heroku_pipelines_config_file
-        build :create_heroku_pipeline
-        build :provide_deploy_script
-        build :configure_automatic_deployment
-      end
+      return unless options[:heroku]
+      say 'Creating Heroku apps'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.